package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "voxa-ga",
-  "version": "2.0.3",
+  "version": "2.1.0-alpha3",
```

Comment:
seems like this was a bug fix, not new functionality, as such should have been 2.0.4 and maybe some alpha instead of 2.1.0

Reply:
add platform specific tracking ID - new functionality
fix geoId on Google Assistant - bug fix
remove health check event - bug fix